storm/check.py

```diff
@@ -205,11 +269,15 @@ def process_cluster_stats(self, environment, cluster_stats):
         :return: Extracted cluster stats metrics
         :rtype: dict
         """
-        if len(cluster_stats) == 0:
-            return {}
-        else:
-            version = _g(cluster_stats, 'unknown', None, 'version').replace(' ', '_').lower()
-            tags = ['stormClusterEnvironment:{}'.format(environment), 'stormVersion:{}'.format(version)]
+        if len(cluster_stats) >= 0:
+            version = _get_string(cluster_stats,
+                                  _get_string(cluster_stats, 'unknown', 'stormVersion'),
```

Comment:
Can stormVersion be something else than a string ? Do we have to use `_get_string` ?